it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -523,9 +526,9 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
 
         stubRetrieveCompanyProfileFromBE(testJourneyId)(status = OK, body = Json.toJsObject(testCioProfile))
         stubStoreIdentifiersMatch(testJourneyId, identifiersMatch = false)(status = OK)
-        stubStoreBusinessVerificationStatus(testJourneyId, BusinessVerificationUnchallenged)(status = OK)
+        stubStoreBusinessVerificationStatus(testJourneyId, BusinessVerificationNotEnoughInformationToCallBV)(status = OK)
```

Comment:
You are stubbing saving BusinessVerificationNotEnoughInformationToCallBV and retrieving the status using businessVerificationNotEnoughInfoToChallengeKey. We could possibly do with a verify check against the business verification status.

Reply:
let me see ....

Reply:
Richard you are right .... it is a mistake and I will use a verify check as you suggested. The test is checking only that the audit is called without checking the content. 
not sure how easy will be but I will try. 
<img width="1322" alt="Screenshot 2022-02-07 at 10 56 19" src="https://user-images.githubusercontent.com/4958044/152775273-06c64bda-55cd-4ad6-8cda-e6ff8a9a6159.png">

Reply:
... but the important line is there ....
        stubStoreBusinessVerificationStatus(testJourneyId, BusinessVerificationNotEnoughInformationToCallBV)(status = OK)
... and that it is important too. let me change it as you suggested.